mod-data-export/mod-data-export.postman_collection.json

```diff
@@ -1282,6 +1585,153 @@
 		{
 			"name": "Cleanup",
 			"item": [
+				{
+					"name": "Delete instance from inventory storage",
```

Comment:
As we are purging the module and then deleting the tenant, deleting the data separately is not necessary